packages/datadog_grpc_interceptor/lib/src/datadog_grpc_interceptor.dart

```diff
@@ -50,10 +60,14 @@ class DatadogGrpcInterceptor extends ClientInterceptor {
     final rumKey = uuid.v1();
     _datadog.rum?.startResourceLoading(
       rumKey,
-      RumHttpMethod.get,
+      RumHttpMethod.unknown,
```

Comment:
RumHttpMethod.unknown actually translates to GET under the hood. I'm removing it for the next release to avoid confusion See #170